tool/e2e/dart_test.yaml

```diff
@@ -0,0 +1,2 @@
+tags:
```

Comment:
I was going to say that this doesn't need tags since it exists only for e2e but I don't think it is worth removing it now

Reply:
I agree with the above.